### ISSUE TITLE
Ukrainian language isn't present in language select menu

### ISSUE THREAD

**Issue by nazarparuna** [HabitRPG/habitica-android#1676]
**Describe the bug**
As far as I know, this translation exists on the Weblate platform. And I also see a directory called "values-uk" in the resource folder. But it is not possible to choose the Ukrainian language from a mobile device.

**To Reproduce**
- Open Habitica on Android
- Click "Settings"
- Click "Language"

**Expected behavior**
"Українська" should be present in language select menu.

**Screenshots**
![Screenshot_2021-11-26-10-05-08-820_com habitrpg android habitica](https://user-images.githubusercontent.com/9466262/143548828-0244a78d-1a77-4b54-b8ba-8872fc8e0a2c.jpg)
![Screenshot_2021-11-26-10-05-10-905_com habitrpg android habitica](https://user-images.githubusercontent.com/9466262/143548867-4ddb30cc-3c2c-497a-b8a9-90d14fb90271.jpg)

**Smartphone (please complete the following information):**
 - Device: Xiaomi Mi 9T Pro
 - OS: Android 11 RKQ1.200826.002
 - App Version: 3.4.1.1 (3091)
 - Habitica Username: @punk-rock

**Comment by alejandro-rios** [comment]
@saraolson Can I work on this?

**Comment by alejandro-rios** [comment]
@nazarparuna one thing I noticed is that it is not fully translated into Ukrainian, not sure if we can include those translations.

<img src="https://user-images.githubusercontent.com/10689052/147889305-43d8036f-fd96-4f8c-80dd-da2faf5d0b3c.png" width="400" />

<img src="https://user-images.githubusercontent.com/10689052/147889311-5bfa5dec-7f70-4872-b443-bf15797571c7.png" width="400" />

**Comment by nazarparuna** [comment]
@alejandro-rios thanks for letting me know. Seems like not all strings are present on Weblate.

**Comment by alejandro-rios** [comment]
There are about 105 strings without translation.

**Comment by ahmetburakbarakli** [comment]
I don't know if this will make things easier or harder but as you can see there exist a Turkish translation but it is not fully completed as well.


<img src="https://user-images.githubusercontent.com/22160644/148103965-afdca277-83dc-4e27-9aa4-73844640bd68.png" alt="drawing" width="300"/> <img src="https://user-images.githubusercontent.com/22160644/148103972-3569e454-a843-4cfe-b06c-50063bf1b1c5.png" alt="drawing" width="300"/>

**Comment by saraolson** [comment]
@alejandro-rios if you'd still like to work on this, go ahead! I'll mark it as in progress. our translations are handled by volunteers, so many translations aren't complete.

**Comment by alejandro-rios** [comment]
@saraolson sure, i'll do it 😄
@nazarparuna can you help me with the missing translations? to include those in the PR

**Comment by nazarparuna** [comment]
@alejandro-rios sure, I can help. But how do you see it?

**Comment by alejandro-rios** [comment]
> @alejandro-rios sure, I can help. But how do you see it?

You can check which are missing in the `Translations Editor` on Android Studio, just click on the first list and search for the one you need
![image](https://user-images.githubusercontent.com/10689052/148289694-e0e722b3-3803-453f-9ece-748a364a37a0.png)

or if you prefer here's the [list](https://gist.github.com/alejandro-rios/e1bddb45da5dd97c572bea7cde594a38)

**Comment by alejandro-rios** [comment]
@ahmetburakbarakli in comparison the list of missing translations for the Turkish language is smaller, if you wish you can make the translations and pass them to me to add them to PR.

**Comment by ahmetburakbarakli** [comment]
@alejandro-rios Sure, but I couldn't understand what to send you after I changed it from the editor. Also, I can see there are missing translation in editor but I couldn't find any string from the screenshot I sent. I used ```develop``` branch. Am I doing something wrong?

**Comment by nazarparuna** [comment]
@alejandro-rios hello
here is the [list](https://gist.github.com/nazarparuna/9d937c534c8536bb46cca86978e11775) of missing translation

**Comment by alejandro-rios** [comment]
@nazarparuna Thank you!! 

@ahmetburakbarakli here's a video on how to get the list of missing translations, you're right, I could not find some of the ones you mention, maybe they come from the API?

https://user-images.githubusercontent.com/10689052/148547061-0792dae1-1d89-4eb2-9ce8-2f06e386d43a.mov

**Comment by ahmetburakbarakli** [comment]
@alejandro-rios Thank you for the detailed explanation! But I think I couldn't explain myself well as that was not the thing I asked. I was trying to ask how to deliver translated strings to you but now that @nazarparuna sent the list, I understand now. I am planning to send two lists one for original strings and one for translated ones similar to . 

Also about that missing strings: I have checked these strings from screenshot in the habitica.com, weblate and the mobile application. There exist translation in under habitica project in weblate and it is displayed translated in habitica.com. However, there is not "not translated" string in habitica android project in weblate and string xmls in res/. I don't necessarily want those sentences translated. These are just my findings and I stopped the investigation.
